app/lib/presenter/feedback.rb

```diff
@@ -4,6 +4,7 @@ class Feedback < BasePresenter
     TYPE = 'UF144908'.freeze
     PARTY_CONTEXT = 'Main'.freeze
 
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Maybe just add this as an exception in the rubocop config? We ignore it multiple places

Reply:
done